sh/tmuxcomplete

```diff
@@ -27,7 +27,8 @@
 # Words including 2000 lines of history per pane
 #     sh tmuxcomplete -c '-S -2000'
 
-if ! tmux info &> /dev/null; then
+tmux info &> /dev/null
+if [ "$?" != "0" ]; then
```

Comment:
Thanks a lot for helping me avoid follow up mistakes. Is there some kind of tool to check for POSIX compatibility?

I will update this tomorrow :+1: